src/all/kavita/build.gradle

```
@@ -0,0 +1,16 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlinx-serialization'
+
+ext {
+    extName = 'Kavita'
+    pkgNameSuffix = 'all.kavita'
+    extClass = '.Kavita'
+    extVersionCode = 6
```

Extension versions should start in 1 if they are new.

```suggestion
    extVersionCode = 1
```